provider.go

```diff
@@ -812,6 +795,10 @@ func (h *host) Call(ctx context.Context, req CallRequest, call comProvider.CallF
 	if err != nil {
 		return CallResponse{}, err
 	}
+
+	// note that req.ReturnDependencies is silently discarded
+	// because the information is simply derived from the property values in r.Return.
+
```

Comment:
If `req.AcceptsOutputValues` is false, we will need to reconstruct `ReturnDependencies` when sending out.

Reply:
That's true in general but not relevant here because in host.Call we're unmarshaling from rpc, not marshaling to rpc.  Later, in provider.Call, the rpc result is constructed.